src/test/java/org/dependencytrack/tasks/repositories/NugetMetaAnalyzerTest.java

```diff
@@ -22,16 +22,11 @@
 import org.dependencytrack.model.Component;
 import org.dependencytrack.model.RepositoryType;
 import org.junit.Assert;
+import org.junit.Test;
 
 public class NugetMetaAnalyzerTest {
 
-    //@Test
-    /*
-     * Noticed that Microsoft NuGet is broken for package 3.13.0, which as of this moment, is the latest
-     * version available. However, the registration3 API returns a 404 when making queries for it and the call
-     * to index.json for the nuget package returns 3.12.0 as the latest version. This is breaking DT unit tests.
-     * Commenting out for now.
-     */
+    @Test
```

Comment:
This test doesn't cover the new `initializeEndpoint` functionality. It'd be great if we had another test that covers it.